tornado-unittests/src/main/java/uk/ac/manchester/tornado/unittests/fails/TestFails.java

```diff
@@ -48,7 +48,7 @@ private void reset() {
         for (int i = 0; i < TornadoRuntime.getTornadoRuntime().getNumBackends(); i++) {
             final TornadoBackend driver = TornadoRuntime.getTornadoRuntime().getBackend(i);
             for (int j = 0; j < driver.getDeviceCount(); j++) {
-                driver.getDevice(j).reset();
+                driver.getDefaultDevice().clean();
```

Comment:
why did we select the default device for all iterations over the `driver.getDeviceCount()`?

Reply:
After looking at this unit-test, this method is called only from `test01` and it should run only for the default device. So, why do we iterate over the number of devices that exist in the driver of all backends?

Reply:
This was a mistake after replicating a change from another test. I will fix this.